gax-httpjson/src/main/java/com/google/api/gax/httpjson/ProtoRestSerializer.java

```diff
@@ -82,7 +82,7 @@ RequestT fromJson(InputStream message, Message.Builder builder) {
       JsonFormat.parser().ignoringUnknownFields().merge(json, builder);
       return (RequestT) builder.build();
     } catch (IOException e) {
-      throw new RuntimeException("Failed to parse response message", e);
+      throw new ProtoRestSerializationException("Failed to parse response message", e);
```

Comment:
definitely checked here

Reply:
Please check the other comment about checked exception.